pandas/tests/series/test_datetime_values.py

```diff
@@ -127,8 +127,14 @@ def compare(s, name):
         tm.assert_series_equal(result, expected)
 
         # datetimeindex with tz
-        s = Series(date_range('20130101', periods=5, tz='US/Eastern'),
-                   name='xxx')
+        values = [
+            '2012-01-01 13:00:00',
```

Comment:
The original values are already given a frequency by constructing with `date_range` so the test that was confirming their frequency wasn't actually doing anything useful.